_extras/checkout.md

```diff
@@ -247,10 +247,11 @@ also email you to let you know, as this information is typically not shared duri
 after you have attended your teaching demonstration.
 
 ## 4. Tracking Your Progress
-After you complete this workshop, your application will be matched to your training record and a profile will be created in The
+After you complete this workshop, your application or profile-creation form will be matched to your training record and used to 
+create a profile that you can access in The
```

Comment:
this is hard to parse.  doe s it work like:
1. application or profile creations form at time of registration
2. workshop
3. training record added so that progress can be monitored

modificaitons can be made (and github linked) by e-mailing

Reply:
yes. Basically AMY can take an application and turn it into a "person" profile where training records are tracked, but this requires us to press buttons, which we don't do until after someone has attended an IT event. Would it help to have numbered steps here?